unity/Assets/Scripts/Editor/Tests/ArtGallery/Util/RotateLineLighthouseToLighthouseVisibilityTest.cs

```diff
@@ -0,0 +1,610 @@
+﻿using Assets.Scripts.ArtGallery.Util;
+
+namespace ArtGallery.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using NUnit.Framework;
+    using UnityEngine;
+    using Util.Geometry;
+    using Util.Geometry.Polygon;
+    using Util.Math;
+    using Util.Algorithms.Polygon;
+
+    [TestFixture]
+    public class RotateLineLighthouseToLighthouseVisibilityTest
```

Comment:
Check out how the `IUnionTests` are done in my branch so you don't have to duplicate the tests

Reply:
Done, Still need to move one thing around and then it follows the same test structure as your tests.